suzieq/poller/controller/source/netbox.py

```diff
@@ -151,6 +164,24 @@ def _token_auth_header(self) -> Dict:
         """
         return {'Authorization': f'Token {self._data.token}'}
 
+    def _get_url_list(self) -> List[str]:
+        """Return the list of requests to execute
+
+        Returns:
+            List[str]: list of urls
+        """
+        urls = []
+        url_address = f'{self._server.protocol}://{self._server.host}:'\
+            f'{self._server.port}/api/dcim/devices/?'
+        for tag in self._data.tag:
+            query = url_address
+            for (i, t) in enumerate(tag.split(", ")):
```

Comment:
This works only when you add a space after the comma: `one, two`. A solution might be splitting by comma and then stripping the remaining spaces, if any:
```suggestion
            match_tags = [t.strip() for t in tag.split(',')]
            for i, t in enumerate(match_tags):
``